### PR TITLE
added language description

### LINKED ISSUE #132: Missing ABOUT.md
The [about page](http://exercism.io/languages/lisp/about) is missing a description. If it's alright, I would like to PR one.

### PR DESCRIPTION
**Pull request by definemethod**
This should fix #132.

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Common Lisp is the most prominent, (together with Scheme) offshoot of the long-running family of Lisp programming languages. 
+It's a multi-paradigm programming language that allows you to to choose the approach and paradigm
```
Comment (yurrriq) [inline docs/ABOUT.md]: `s/to to/to`

**Comment by verdammelt** [comment]
Perhaps along with Scheme, Clojure should be mentioned. It is currently probably *the* most prominent Lisp.

**Comment by definemethod** [comment]
I went ahead and removed the reference to Scheme altogether. Mentioning both Scheme and Clojure made the description a bit cluttered in my opinion. I also removed the extra "to".

**Comment by verdammelt** [comment]
This sounds good. It gets my vote 👍 

I'll wait a little while and then merge unless @yurrriq has any more comments/issues.

**Comment by yurrriq** [comment]
LGTM

**Pull request merged**